.gitmodules

```diff
@@ -0,0 +1,15 @@
+[submodule "modules/core"]
+	path = modules/core
+	url = https://bitbucket.org/notabots/core.git
+[submodule "modules/customCommands/data"]
+	path = modules/customCommands/data
+	url = https://bitbucket.org/notabots/customcommands.git
+[submodule "modules/notAchili/data"]
+	path = modules/notAchili/data
+	url = https://bitbucket.org/notabots/notachili.git
```

Comment:
Joe, regarding the investigation - what is the scope or the method? maybe i can provide data myself?

During experiments (while refactoring) I had game running with 4 instances of chili at the same time and i have not reached any gap. In profiling there were no significat spikes. Is your RAM fear based on some real experience?

Chili objects can consume few KBs and images of the UI few MBs at the most. (whole folder have 703 KB) :)

Reply:
Don't worry. I'll just check we are not in a critical situation. In such a case, we can merge and I'll further investigate (eventaully opening issues or additional PRs)

Reply:
ok, thank you.

Reply:
OK, it seems we are safe! the GUI does not seems to blow up the LUA memory over 200MB. Another question to consider in the future is where the hell is going the extra consumed memory? :-p

Reply:
Good, regarding this topic I understand it as a green light. I never observed huge memory consumption unless someone did some procedural magic which got out from expected complexity/size bounds. Or potentially some lame widgets using chili can create not infinit but growing recursion on chili objects (e.g. on each click instead of updating parent you create new child object).

Reply:
The memory issue is closed. We can further investigate after integration. But is not a green light, we have the license issues